jupyterlab_git/handlers.py

```diff
@@ -647,7 +647,7 @@ async def post(self, path: str = ""):
         POST get (if no options are passed) or set configuration options
         """
         data = self.get_json_body()
```

Comment:
I would set the default here. That will prevent future error if more information are looked for in data.

```suggestion
        data = self.get_json_body() or {}
```